carbon-design-system/carbon-addons-iot-react
# 3234 table create manage views story

Closes #3234

**Summary**
Created story group for Manage views. This PR also solves #3243 

**Change List (commits, features, bugs, etc)**
- Create new story group
- Update/improve related stories 
- Create new mdx files, write docs and link from stories

**Acceptance Test (how to verify the PR)**
- Test that the table example at https://deploy-preview-3246--carbon-addons-iot-react.netlify.app?path=/story/1-watson-iot-table-user-view-management--with-user-view-management works as expected 
- Verify that the issue described in #3243 also is resolved
- Verify that the story links to to the proper mdx
- Verify that the mdx makes sense and that the links work
- Verify the component stories and docs 
  - https://deploy-preview-3246--carbon-addons-iot-react.netlify.app?path=/docs/1-watson-iot-table-user-view-management-tablemanageviewsmodal--with-callbacks-implemented
  - https://deploy-preview-3246--carbon-addons-iot-react.netlify.app?path=/docs/1-watson-iot-table-user-view-management-tablesaveviewmodal--playground
  - https://deploy-preview-3246--carbon-addons-iot-react.netlify.app?path=/docs/1-watson-iot-table-user-view-management-tableviewdropdown--playground

**Regression Test (how to make sure this PR doesn't break old functionality)**

- tests here



**Things to look for during review**
